[2023-09-22T23:36:55.744+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: create_multi_table.create_users_table scheduled__2023-09-01T00:00:00+00:00 [queued]>
[2023-09-22T23:36:55.763+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: create_multi_table.create_users_table scheduled__2023-09-01T00:00:00+00:00 [queued]>
[2023-09-22T23:36:55.763+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-22T23:36:55.764+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-22T23:36:55.765+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-22T23:36:55.787+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_users_table> on 2023-09-01 00:00:00+00:00
[2023-09-22T23:36:55.792+0000] {standard_task_runner.py:55} INFO - Started process 172 to run task
[2023-09-22T23:36:55.795+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'create_multi_table', 'create_users_table', 'scheduled__2023-09-01T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/crt_tbls_trs_data.py', '--cfg-path', '/tmp/tmp_xg4ekk2']
[2023-09-22T23:36:55.796+0000] {standard_task_runner.py:83} INFO - Job 34: Subtask create_users_table
[2023-09-22T23:36:55.875+0000] {task_command.py:376} INFO - Running <TaskInstance: create_multi_table.create_users_table scheduled__2023-09-01T00:00:00+00:00 [running]> on host c71b9a166a04
[2023-09-22T23:36:55.940+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_multi_table
AIRFLOW_CTX_TASK_ID=create_users_table
AIRFLOW_CTX_EXECUTION_DATE=2023-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-01T00:00:00+00:00
[2023-09-22T23:36:55.950+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-09-22T23:36:55.955+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
                first_name TEXT NULL,
                last_name TEXT NULL,
                email TEXT NULL,
                street_number SMALLINT NULL,
                street_name TEXT NULL,
                surburb TEXT NULL,
                post_code INTEGER NULL,
                state TEXT NULL,
                country TEXT NULL
            );
        , parameters: None
[2023-09-22T23:36:55.956+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-09-22T23:36:55.967+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=create_multi_table, task_id=create_users_table, execution_date=20230901T000000, start_date=20230922T233655, end_date=20230922T233655
[2023-09-22T23:36:56.007+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-22T23:36:56.038+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
